src/silicone/stats.py

```diff
@@ -103,8 +103,7 @@ def rolling_window_find_quantiles(
 
 def calc_all_emissions_correlations(emms_df, years, output_dir):
     """
-    Save csv files of the correlation coefficients and the rank correlation coefficients
-     between emissions at specified times.
+    Save csv files of the correlation coefficients and the rank correlation coefficients between emissions at specified times.
```

Comment:
Not really sure why this is helpful? Just makes things harder to read

Reply:
Oh right that was my attempt to fix the docs, compare https://silicone.readthedocs.io/en/latest/stats.html#silicone.stats.calc_all_emissions_correlations with the attached screenshot. 
![Screen Shot 2020-05-04 at 7 45 37 am](https://user-images.githubusercontent.com/23625371/80926533-439df100-8ddb-11ea-818d-705f31564034.png)

I've found a better solution though (turns out you only need to delete the space, not the newline), just pushed.

Reply:
Great, thanks!